src/internal/clusterstate/v2.8.0/pfsdb.go

```diff
@@ -308,19 +327,155 @@ func createNotifyCommitsTrigger(ctx context.Context, tx *pachsql.Tx) error {
 
 }
 
-// Migrate commits from collections.commits to pfs.commits
 func migrateCommitSchema(ctx context.Context, env migrations.Env) error {
 	if err := alterCommitsTable(ctx, env.Tx); err != nil {
 		return err
 	}
 	if err := createCommitAncestryTable(ctx, env.Tx); err != nil {
 		return err
 	}
-	// todo(fahad): migrate commits
-	// todo(fahad): call createNotifyCommitsTrigger() once migration is complete
 	return nil
 }
 
+type commitCollection struct {
+	v2_7_0.CollectionRecord
+	IntID uint64 `db:"int_id"`
+}
+
+func migrateCommitsFromCollections(ctx context.Context, env migrations.Env) error {
+	count := struct {
+		Collections uint64 `db:"col_count"`
+		Commits     uint64 `db:"commits_count"`
+	}{}
+	rowCountQuery := `SELECT count(commit.int_id) AS commits_count, count(col.key) 
```

Comment:
nit: for queries that are only executed once like this, I prefer to inline them in the function call.